route/route-free-trial-request.js

```diff
@@ -47,6 +47,7 @@ module.exports = router => {
       const postResponse = {};
       const studentAvailability = makeAvailability(availability);
 
+      req.body.instrument.toUpperCase();
```

Comment:
Make sure and reassign, eg, ```req.body.instrument = req.body.instrument.toUpperCase();```

Reply:
I feel silly. Added the reassignment var.